mlflow/gateway/providers/openai.py

```diff
@@ -0,0 +1,151 @@
+import openai
+from fastapi.encoders import jsonable_encoder
+
+from .base import BaseProvider
+from ..schemas import chat, completions, embeddings
+
+
+class OpenAIProvider(BaseProvider):
+    NAME = "openai"
+    SUPPORTED_ROUTES = ("chat", "completions", "embeddings")
+
+    async def chat(self, payload: chat.RequestPayload) -> chat.ResponsePayload:
+        resp = await openai.ChatCompletion.acreate(
+            model=self.config.model.name,
+            messages=jsonable_encoder(payload.messages),
```

Comment:
```suggestion
            messages=jsonable_encoder(payload.messages),
            temperature=...
```

Still need to add parameters.